cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -264,291 +415,234 @@ datasource "ds_describe_snapshots" do
       field "startTime", xpath(col_item, "startTime")
       field "ownerId", xpath(col_item, "ownerId")
       field "volumeSize", xpath(col_item, "volumeSize")
+      field "service", "EBS"
     end
   end
 end
 
-datasource "ds_filter_old_snapshots" do
-  run_script $js_filter_old_snapshots, $snapshot_age, $ds_describe_snapshots, $param_exclude_tags
+script "js_describe_snapshots", type: "javascript" do
+  result "results"
+  parameters "region", "ds_get_caller_identity"
+  code <<-EOS
+    results = {
+        "auth": "auth_aws",
```

Comment:
Line 428-444 - minor thing but I think the tab/spaces is set to 4 rather than 2

Reply:
Can you clarify what you're looking for please?

```ruby
  code <<-EOS
  // Here is some JS code
  var foo = "bar";
  return foo;
  EOS
```
or
```ruby
  code <<-EOS
    // Here is some JS code
    var foo = "bar";
    return foo;
  EOS
```
or
```ruby
  code <<-EOS
// Here is some JS code
var foo = "bar";
return foo;
  EOS
```

Whatever we decide, perhaps we consider adding a rule to the Dangerfile as well to test against this, and start normalizing the rest of the PTs too over time (until we can get some sort of include or package functionality)

Thank you!